### PR TITLE
Split views

### PR DESCRIPTION
**Pull request by jdungan**

### CODE REVIEW

File: api/urls.py [hunk 1]
```diff
@@ -13,21 +13,9 @@
...
-    url(r'^bid-list', views.BidList.as_view(), name='bid-list'),
-    url(r'^claim-list', views.ClaimList.as_view(), name='claim-list'),
-    url(r'^vote-list', views.VoteList.as_view(), name='vote-list'),
-    url(r'^payout-request',
-        views.PayoutViewSet.as_view(), name='payout-request'),
-)
 
 urlpatterns = patterns(
     '',
-    url(r'^', include(custom_endpoint_url_patterns, namespace="custom-urls")),
```
Comment (groovecoder) [inline api/urls.py]: happydance

**Comment by groovecoder** [comment]
After I voted to approve a claim, I was sent to /votes/ which was the default DRF view ...

codesy-vote-list

**Comment by groovecoder** [comment]
That was the only issue I saw. Want to fix before merge, or after?

**Comment by jdungan** [comment]
Oops.  That's ugly.

**Comment by jdungan** [comment]
Why does codecov hate me?

**Pull request merged**